test/envoye2e/env/tcp_envoy_conf.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+const tcpEnvoyClientConfTemplYAML = `
+admin:
+  access_log_path: {{.ClientAccessLogPath}}
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: {{.Ports.ClientAdminPort}}
+static_resources:
+  clusters:
+  - name: client
+    connect_timeout: 5s
+    type: STATIC
+    hosts:
```

Comment:
`hosts` is deprecated, please update the config file. I think the new way is similar to EDS load assignment API.

Reply:
Fixed.. also fixed it in envoy_conf.go